.circleci/config.yml

```diff
@@ -107,13 +107,36 @@ jobs:
           paths:
             - ~/.gradle
           key: gradle-{{ checksum "build.gradle" }}
+# Integration tests
+  integration-tests:
+    environment:
+      # Configure the JVM and Gradle to avoid OOM errors
+      _JAVA_OPTIONS: "-Xmx1g"
+      GRADLE_OPTS: "-Dorg.gradle.daemon=false -Dorg.gradle.workers.max=2"
+    docker:
+      - image: openkbs/jdk-mvn-py3
```

Comment:
this docker image has pip3, but output shows that pip3 not found.
how it's possible?

Reply:
maybe it's not on the default PATH?

Reply:
no, something different